### PR TITLE
Sample app now for ios and android

### PR DESCRIPTION
**Pull request by wmathurin**
App is also richer than before.

(Still investigating a bug on iOS - after logout - app is in a bad state - and only works correctly if restarted)

### CODE REVIEW

File: app/android/app/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+				<action android:name="android.intent.action.MAIN" />
+				<category android:name="android.intent.category.LAUNCHER" />
+			</intent-filter>
+		</activity>
+
+        <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
+
+	</application>
+
+    <uses-sdk android:minSdkVersion="17"
```
Comment (bhariharan) [inline app/android/app/AndroidManifest.xml]: Let's change this to `minSdkVersion="19"`, to be consistent with the `KitKat` minimum update on `unstable`.

File: app/ios/PodFile [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '6.0'
```
Comment (bhariharan) [inline app/ios/PodFile]: Shouldn't this be `8.0` minimum?

**Comment by bhariharan** [comment]
Just those few minor comments. Looks good otherwise.

**Comment by wmathurin** [comment]
Aligned min versions with SDK https://github.com/wmathurin/SmartSyncExplorerReactNative/commit/aa119d13695fc0e4aab87484d5fd66bffa5b0004

**Pull request merged**